dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsHelper.java

```diff
@@ -136,75 +138,73 @@ Response getServiceIntegrationSiteView(
                         paginationContext.getOrderBy(),
                         orderDirection,
                         Collections.emptyMap(),
-                        (Function<PaginatedArrayList<SiteView>, ServiceIntegrationView>) paginatedArrayList -> {
+                        (Function<PaginatedArrayList<SiteView>, AppView>) paginatedArrayList -> {
                             final long count = sitesWithConfigurations.size();
-                            return new ServiceIntegrationView(serviceDescriptor, count,
+                            return new AppView(appDescriptor, count,
                                     paginatedArrayList);
                         });
     }
 
     /**
-     * This gives you a detailed view with all the configuration secrets for a given service-key host pair.
-     * @param serviceKey unique service id for the given host.
+     * This gives you a detailed view with all the configuration secrets for a given appKey host pair.
+     * @param key unique app id for the given host.
      * @param siteId Host Id
      * @param user Logged in user
      * @return view
      * @throws DotSecurityException
      * @throws DotDataException
      */
-    Optional<ServiceIntegrationView> getServiceIntegrationSiteDetailedView(
-            final String serviceKey, final String siteId,
+    Optional<AppView> getAppSiteDetailedView(
+            final String key, final String siteId,
             final User user)
             throws DotSecurityException, DotDataException {
-        final Optional<ServiceDescriptor> serviceDescriptorOptional = serviceIntegrationAPI
-                .getServiceDescriptor(serviceKey, user);
-        if (serviceDescriptorOptional.isPresent()) {
-            final ServiceDescriptor serviceDescriptor = serviceDescriptorOptional.get();
+        final Optional<AppDescriptor> appDescriptorOptional = appsAPI
+                .getAppDescriptor(key, user);
+        if (appDescriptorOptional.isPresent()) {
+            final AppDescriptor appDescriptor = appDescriptorOptional.get();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)